libraries/chain/son_wallet_evaluator.cpp

```diff
@@ -72,6 +72,7 @@ object_id_type update_son_wallet_evaluator::do_apply(const son_wallet_update_ope
       if (itr->addresses.find(op.sidechain) == itr->addresses.end()) {
          db().modify(*itr, [&op](son_wallet_object &swo) {
             swo.addresses[op.sidechain] = op.address;
+            swo.redeem_scripts[op.sidechain] = op.redeem_script;
```

Comment:
Redeem sctipt is already stored in address field, as address and a script are tightly paired. This field is not needed and breaks abstraction, by adding bitcoin specific field.

Reply:
ok